<html>
	<head>
        <!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-36889698-2"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-36889698-2');
</script>

		<link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width,initial-scale=1">
		<title>The COVID-19 Testing Group</title>
	</head>

    <body>
        <div class="container">
            
            <div class="section">
                <div class="center-align">
                    <a href="/covid19testgroup">
                    <img src="/assets/images/covid/header.png" style="margin-bottom:-50px">
                    </a>
                </div>
                <h1 class="center-align">COVID-19 Testing Group</h1>
            </section>

<div class="divider"></div>
            <!-- <div style="background-color: #fad6d8" class="center-align"><strong>This community needs your input! Suggest a paper, study, Twitterthread, or resource <a href="https://forms.gle/P1AWBK1F9ibiqzRr8">here</a>.</strong></div> -->

            <div class="section">
                <div>The COVID-19 Testing Group is a community resource for <strong>sharing the latest information on COVID-19 testing, prevalence, seroprevalence, and burden studies, planning tools, and data.</strong> This website is maintained by the <a href="https://larremorelab.github.io">Larremore Lab</a> at the University of Colorado Boulder with collaborations in the <a href="https://www.hsph.harvard.edu/yonatan-grad/">Grad Lab</a> and the <a href="https://ccdd.hsph.harvard.edu/people/michael-mina/">Mina Lab</a> at the Harvard T.H. Chan School of Public Health, and <a href="https://www.jkbrennan.com/">Brennan Klein</a> at Northeastern University. 
                </div>
            </div>


<h2 class="center-align">COVID Test Delays</h2>
            <div class="row">
                <div class="col s12 m4">
                    <div class="card medium">
                        <div class="card-image" style="max-height:75%; padding-top: 10px;">
                            <a href="/assets/images/covid/delays_sorted_0729.pdf">
                                <img src="/assets/images/covid/delays_sorted_0729.png">
                            </a>
                        </div>
                        <div class="card-content" style="max-height:20%">
                            <h5>View the <a href="/assets/images/covid/delays_sorted_0729.pdf">results</a></h5>
                        </div>
                    </div>
                </div>

                <div class="col s12 m4">
                    <div class="card medium">
                        <div class="card-image" style="max-height:75%; padding-top: 10px;">
                            <a href="http://tinyurl.com/testdelaysurvey">
                                <img src="/assets/images/covid/survey.png">
                            </a>
                        </div>
                        <div class="card-content" style="max-height:20%">
                            <h5>Take the <a href="http://tinyurl.com/testdelaysurvey">survey</a></h5>
                        </div>
                    </div>
                </div>

                <div class="col s12 m4">
                    <div class="card medium">
                        <div class="card-image" style="max-height:75%; padding-top: 10px;">
                            <a href="http://tinyurl.com/testdelaydata">
                                <img src="/assets/images/covid/data.png">
                            </a>
                        </div>
                        <div class="card-content" style="max-height:20%">
                            <h5>Get the <a href="http://tinyurl.com/testdelaydata">data</a></h5>
                        </div>
                    </div>
                </div>

            </div>

<h2 class="center-align">Planning & Analysis Tools</h2> 
            <div class="row">
                <div class="col s12 m4">
                    <div class="card large">
                        <div class="card-image">
                            <a href="/covid-calculator3">
                                <img src="/assets/images/covid/calc3.png">
                            </a>
                        </div>
                        <div class="card-content" style="max-height:60%">
                            <h5>Surveillance policy calculator</h5>
                            <div>Estimate the impact of test frequency, turnaround time, and limit of detection, on the effectiveness of screening and surveillance testing for SARS-CoV-2.</div>
                        </div>
                    </div>
                </div>

                <div class="col s12 m4">
                    <div class="card large">
                        <div class="card-image">
                            <a href="/covid-calculator1">
                                <img src="/assets/images/covid/calc1.png">
                            </a>
                        </div>
                        <div class="card-content" style="max-height:60%">
                            <h5>Prevalence calculator: known se and sp</h5>
                            <p>Calculate Bayesian posterior distributions for prevalence when sensitivity and specificity are known.</p>
                        </div>
                    </div>
                </div>

                <div class="col s12 m4">
                    <div class="card large">
                        <div class="card-image">
                            <a href="/covid-calculator2">
                                <img src="/assets/images/covid/calc2.png">
                            </a>
                        </div>
                        <div class="card-content" style="max-height:60%">
                            <h5>Prevalence calculator for tests with lab calibration data</h5>
                            <p>Calculate Bayesian <em>joint</em> posterior distributions for prevalence, se, and sp.</p>
                        </div>
                    </div>
                </div>

            </div>

<h2 class="center-align">Related Papers</h2> 

<ul class="collapsible expandable">
        <li>
            <div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Test sensitivity is secondary to frequency and turnaround time for COVID-19 surveillance</div>
            <div class="collapsible-body"><strong>Larremore</strong> et al. 2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.06.22.20136309v2">link</a>]<br><strong>Summary — </strong>The COVID-19 pandemic has created a public health crisis. Because SARS-CoV-2 can spread from individuals with pre-symptomatic, symptomatic, and asymptomatic infections, the re-opening of societies and the control of virus spread will be facilitated by robust surveillance, for which virus testing will often be central. After infection, individuals undergo a period of incubation during which viral titers are usually too low to detect, followed by an exponential growth of virus, leading to a peak viral load and infectiousness, and ending with declining viral levels and clearance. Given the pattern of viral load kinetics, we model surveillance effectiveness considering test sensitivities, frequency, and sample-to-answer reporting time. These results demonstrate that effective surveillance, including time to first detection and outbreak control, depends largely on frequency of testing and the speed of reporting, and is only marginally improved by high test sensitivity. We therefore conclude that surveillance should prioritize accessibility, frequency, and sample-to-answer time; analytical limits of detection should be secondary.</div></li>
        <li>
            <div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Jointly modeling prevalence, sensitivity and specificity for optimal sample allocation</div>
            <div class="collapsible-body"><strong>Larremore, Fosdick, Zhang, Grad</strong> 2020 [<a href="https://www.biorxiv.org/content/10.1101/2020.05.23.112649v1">link</a>]<br><strong>Summary — </strong>The design and interpretation of prevalence studies rely on point estimates of the performance characteristics of the diagnostic test used. When the test characteristics are not well defined and a limited number of tests are available, such as during an outbreak of a novel pathogen, tests can be used either for the field study itself or for additional validation to reduce uncertainty in the test characteristics. Because field data and validation data are based on finite samples, inferences drawn from these data carry uncertainty. In the absence of a framework to balance those uncertainties during study design, it is unclear how best to distribute tests to improve study estimates. Here, we address this gap by introducing a joint Bayesian model to simultaneously analyze lab validation and field survey data. In many scenarios, prevalence estimates can be most improved by apportioning additional effort towards validation rather than to the field. We show that a joint model provides superior estimation of prevalence, as well as sensitivity and specificity, compared with typical analyses that model lab and field data separately.</div></li>
        <li>
            <div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Implications of test characteristics and population seroprevalence on ‘immune passport’ strategies</div>
            <div class="collapsible-body"><strong>Larremore, Bubar, Grad</strong> 2020 [<a href="https://dash.harvard.edu/handle/1/42664007">link</a>]<br><strong>Summary — </strong>Social distancing and other community quarantine measures have slowed the spread of COVID-19 but have also contributed to an economic shutdown with immense cost and growing pressures to return people to work. Among various strategies, one is the use of “immune passports”, which would allow individuals with serological evidence of exposure to SARS-CoV-2 to return to work. This is premised on the belief that antibodies confer sufficient immunity to prevent COVID-19 infection, and carries both ethical and scientific challenges.</div></li>
        <li>
            <div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Estimating SARS-CoV-2 seroprevalence and epidemiological parameters with uncertainty from serological surveys</div>
            <div class="collapsible-body"><strong>Larremore</strong> et al. 2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.04.15.20067066v2">link</a>]<br><strong>Summary — </strong>Establishing how many people have already been infected by SARS-CoV-2 is an urgent priority for controlling the COVID-19 pandemic. Patchy virological testing has hampered interpretation of confirmed case counts, and unknown rates of asymptomatic and mild infections make it challenging to develop evidence-based public health policies. Serological tests that identify past infection can be used to estimate cumulative incidence, but the relative accuracy and robustness of various sampling strategies has been unclear. Here, we used a flexible framework that integrates uncertainty from test characteristics, sample size, and heterogeneity in seroprevalence across tested subpopulations to compare estimates from sampling schemes. Using the same framework and making the assumption that serological positivity indicates immune protection, we propagated these estimates and uncertainty through dynamical models to assess the uncertainty in the epidemiological parameters needed to evaluate public health interventions. We examined the relative accuracy of convenience samples versus structured surveys to estimate population seroprevalence, and found that sampling schemes informed by demographics and contact networks outperform uniform sampling. The framework can be adapted to optimize the design of serological surveys given particular test characteristics and capacity, population demography, sampling strategy, and modeling approach, and can be tailored to support decision-making around introducing or removing interventions.</div></li>
        </ul>

<h2 class="center-align">Other Resources</h2> 

<h4>Screening & Surveillance</h4>

<p>Mike was on <a href="https://www.microbe.tv/twiv/twiv-640/">This Week in Virology</a> to talk about how we can break the back of this thing with cheap tests and fast turnaround times.</p>

<p>The MGGG Team at Tufts made a nice <a href="https://mggg.github.io/uni-calculator/">university scenario planner</a>.</p>

<h4>SARS-CoV-2 Serology</h4>

<p><a href="http://twitter.com/DrJarlov">Henrik Jarlov</a> has compiled a comprehensive list of serological studies worldwide, available in a <a href="https://docs.google.com/spreadsheets/d/17Tf1Ln9VuE5ovpnhLRBJH-33L5KRaiB3NhvaiF3hWC0/edit#gid=0">Google Sheet</a>.</p>

<p>The <a href="https://covidtracking.com/">COVID Tracking Project</a> has aggregated data on serological testing into <a href="http://airtable.com/shrGVEvpWp12RyLAI">an Airtable</a>.</p>

<p>FindDx has created a <a href="https://finddx.shinyapps.io/COVID19DxData/">test kit roundup</a>, scattered by sensitivity and specificity. Hover over a data point to learn about the test and its calibration data.</p>
            
            
            
<!--             
<h1 class="center-align">Literature</h1> 
        <div class="center-align">
            <img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 4px; margin-left: 16px;">
            Peer Reviewed
            <img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 4px; margin-left: 16px;">
            Preprint
            <img src="assets/images/covid/li_news.png" height="16" width="16" style="margin-right: 4px; margin-left: 16px;">
            News Media
        </div>
<h4>Seroprevalence studies & outcomes</h4>
	<ul class="collapsible expandable">
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Serological tests facilitate identification of asymptomatic SARS‐CoV‐2 infection in Wuhan, China</div>
			<div class="collapsible-body"><strong>Wu</strong> et al. 4/20/2020 [<a href="https://onlinelibrary.wiley.com/doi/10.1002/jmv.25904">link</a>]<br><strong>Summary — </strong>We observed a ~10% SARS‐CoV‐2‐specific IgG positive rate from 1,402 tests from one hospital in Wuhan. Large‐scale investigation is required to evaluate the herd immunity of the city, for the resuming people and for the re‐opened city.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">COVID-19 Antibody Seroprevalence in Santa Clara County, California</div>
			<div class="collapsible-body"><strong>Bendavid</strong> et al. 4/17/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.04.14.20062463v1">link</a>]<br><strong>Summary — </strong>We measured the seroprevalence of antibodies to SARS-CoV-2 in Santa Clara County. The unadjusted prevalence of antibodies to SARS-CoV-2 in Santa Clara County was 1.5% (exact binomial 95CI 1.11-1.97%), and the population-weighted prevalence was 2.81% (95CI 2.24-3.37%). These results suggest an infection rate that would be 50 to 85 times higher than the official number of confirmed cases.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_news.png" height="16" width="16" style="margin-right: 10px;">Miami-Dade has tens of thousands of missed coronavirus infections, UM survey finds</div>
			<div class="collapsible-body"><strong>Conarck</strong> et al. 4/24/2020 [<a href="https://www.miamiherald.com/news/coronavirus/article242260406.html">link</a>]<br><strong>Summary — </strong>About 6 percent of Miami-Dade’s population — about 165,000 residents — have antibodies indicating a past infection by the novel coronavirus, dwarfing the state health department’s tally of about 10,600 cases, according to preliminary study results announced by University of Miami researchers Friday. This study will be an ongoing weekly survey based on antibody tests. The researchers say they are 95% certain that the true amount of infection lies between 4.4% and 7.9% of the population, with 6% representing the best estimate. Compared to the Santa Clara survey that found an infection rate 50 to 85 times higher than the official number of confirmed cases, UM’s more randomized study indicates an infection rate about 15 times higher than the official case count.</div></li></li>
	</ul>
<h4>Antibody response & Immunity</h4>
	<ul class="collapsible expandable">
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Molecular and serological investigation of 2019-nCoV infected patients: implication of multiple shedding routes</div>
			<div class="collapsible-body"><strong>Zhang</strong> et al. 2/17/2020 [<a href="https://www.tandfonline.com/doi/full/10.1080/22221751.2020.1729071">link</a>]<br><strong>Summary — </strong>Testing for 2019-nCoV infection is via oral swabs - assuming the virus was transmitted through the respiratory tract. We found presence of 2019-nCoV in anal swabs and blood as well, and more anal swab positives than oral swab positives in a later stage of infection, suggesting shedding and thereby transmitted through oral–fecal route. 

With respect to serology, we also showed serology test can improve detection positive rate thus should be used in future epidemiology. On day 5, an increase of viral antibodies can be seen in nearly all patients.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Antibody responses to SARS-CoV-2 in patients of novel coronavirus disease 2019 </div>
			<div class="collapsible-body"><strong>Zhao</strong> et al. 3/28/2020 [<a href="https://academic.oup.com/cid/advance-article/doi/10.1093/cid/ciaa344/5812996">link</a>]<br><strong>Summary — </strong>This paper characterizes the antibody response to SARS-CoV-2 and reports seroconversion time and rates. The seroconversion rate among 173 patients for Ab, IgM and IgG was 93.1%, 82.7% and 64.7%, respectively, with median seroconverstion time at day 11, 12,  and 14, respectively. 

Combining RNA and antibody detections significantly improved the sensitivity of diagnosis for COVID-19. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Severe Acute Respiratory Syndrome Coronavirus 2−Specific Antibody Responses in Coronavirus Disease 2019 Patients</div>
			<div class="collapsible-body"><strong>Okba</strong> et al. 4/8/2020 [<a href="https://wwwnc.cdc.gov/eid/article/26/7/20-0841_article">link</a>]<br><strong>Summary — </strong>We developed serologic assays for detection of SARS-CoV-2 antibodies.  The IgA-based ELISA showed higher sensitivity than the IgG-based ELISA, but the IgG ELISA showed higher specificity. Although the IgA and IgG assays can be used for serologic diagnosis, IgG is longer lived and thus is preferred for serosurveillance studies.

Most PCR-confirmed SARS-CoV-2–infected persons seroconverted by 2 weeks after disease onset</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Antibodies in Infants Born to Mothers With COVID-19 Pneumonia</div>
			<div class="collapsible-body"><strong>Zeng</strong> et al. 3/26/2020 [<a href="https://jamanetwork.com/journals/jama/fullarticle/2763854">link</a>]<br><strong>Summary — </strong>A previous study showed that COVID-19 infection did not transmit from mother to infant. This paper shows that virus-specific antibodies were detected in neonatal blood sera samples. The IgG concentrations were elevated in 5 out of 6 infants  and IgM was detected in 2 out of 6 infants. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Diagnostic value and dynamic variance of serum antibody in coronavirus disease 2019</div>
			<div class="collapsible-body"><strong>Jin</strong> et al. 4/3/2020 [<a href="https://pubmed.ncbi.nlm.nih.gov/32251798/">link</a>]<br><strong>Summary — </strong>Viral serological testing is an effective means of diagnosis for SARS-CoV-2 infection. The positive rate and titer variance of IgG are higher than those of IgM in COVID-19.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Developing antibody tests for SARS-CoV-2</div>
			<div class="collapsible-body"><strong>Petherick</strong> et al. 4/4/2020 [<a href="https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30788-1/fulltext">link</a>]<br><strong>Summary — </strong>General article describing the need for antibody tests and different considerations that need to be taken into account when assessing antibody tests. It also describes how reinfection of SARS-CoV-2 is unlikely because the spike protein is highly conserved. 
</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Generation of antibodies against COVID-19 virus for development of diagnostic tools</div>
			<div class="collapsible-body"><strong>Li</strong> et al. 2/27/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.02.20.20025999v2">link</a>]<br><strong>Summary — </strong>We designed and synthesized peptides as the
immunogens to quickly generate both polyclonal and monoclonal antibodies for development of diagnostic tools against the emerging diseases. Analysis included Western Blots and sandwich ELISA. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Clinical significance of IgM and IgG test for diagnosis of highly suspected COVID-19 infection</div>
			<div class="collapsible-body"><strong>Jia</strong> et al. 3/12/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.02.28.20029025v2">link</a>]<br><strong>Summary — </strong>This paper evaluates  the diagnostic value of IgM and IgG. The positive detection rate of combination of IgM and IgG for patients with COVID-19 negative and positive nucleic acid test was 72.73% and 87.50%. The results were significantly higher than the nucleic acid or IgM, IgG single detection. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Heat inactivation of serum interferes with the immunoanalysis of antibodies to SARS-CoV-2</div>
			<div class="collapsible-body"><strong>Hu</strong> et al. 3/16/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.12.20034231v1">link</a>]<br><strong>Summary — </strong>Heat inactivation, done prior to testing, significantly interferes with the levels of antibodies to SARS-CoV-2. 44.12% of the detected IgM levels were dropped below the cut-off value after heating, suggesting heat inactivation can lead to false-negative results of these samples. Therefore, heat inactivation prior to immunoanalysis is not recommended. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</div>
			<div class="collapsible-body"><strong>Long</strong> et al. 3/20/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.18.20038018v1">link</a>]<br><strong>Summary — </strong>The positive rate for IgG reached 100% around 20 days after symptoms onset. The median day of seroconversion for both lgG and IgM was 13 days after symptoms onset. Seroconversion of IgM occurred at the same time, or earlier, or later than that of IgG. The criteria of IgG seroconversion and ≥ 4-fold increase in the IgG titers in sequential samples together diagnosed 82.9% (34/41) of the patients.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Viral Kinetics and Antibody Responses in Patients with COVID-19</div>
			<div class="collapsible-body"><strong>Tan</strong> et al. 3/26/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.24.20042382v1">link</a>]<br><strong>Summary — </strong>Nasopharyngeal, sputum and stools rather than blood and urine, were the major shedding routes for SARS-CoV-2, and meanwhile sputum had a prolonged viral shedding. Symptom cough seems to be aligned with viral shedding in clinical respiratory and fecal specimens. Stronger antibody response was associated with delayed viral clearance and disease severity.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Global profiling of SARS-CoV-2 specific IgG/ IgM responses of convalescents using a proteome microarray</div>
			<div class="collapsible-body"><strong>Jiang</strong> et al. 3/27/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.20.20039495v1">link</a>]<br><strong>Summary — </strong>This study presents the first whole picture of the SARS-CoV-2 specific IgG/ IgM responses using a SARS-CoV-2 proteome microarray with 18 out of the 28 predicted proteins to profile the IgG/ IgM responses. The results suggest that at the convalescent phase 100% of patients had IgG/ IgM responses to SARS-CoV-2, especially to protein N, S1 but not S2. S1 purified from mammalian cell demonstrated the highest performance to differentiate COVID-19 patients from controls. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Serology characteristics of SARS-CoV-2 infection since the exposure and post symptoms onset</div>
			<div class="collapsible-body"><strong>Lou</strong> et al. 3/27/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.23.20041707v1">link</a>]<br><strong>Summary — </strong>This paper describes the antibody dynamics during SARS-CoV-2 infection. The seroconversion rate for Ab, IgM and IgG in COVID-19 patients was 98.8% (79/80), 93.8% (75/80) and 93.8% (75/80), respectively. The first detectible serology marker is total antibody and followed by IgM and IgG, with a median seroconversion time of 15, 18 and 20 day post exposure (d.p.e) or 9, 10 and 12 days post onset, separately. The sensitivities of Ab, IgM and IgG detection increased to 100%, 96.7% and 93.3% two weeks later, respectively. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">SARS-CoV-2 proteome microarray for mapping COVID-19 antibody interactions at amino acid resolution</div>
			<div class="collapsible-body"><strong>Wang</strong> et al. 3/28/2020 [<a href="https://www.biorxiv.org/content/10.1101/2020.03.26.994756v1">link</a>]<br><strong>Summary — </strong>We created a SARS-CoV-2 proteome microarray to analyze antibody interactions at amino acid resolution. This array reveals that antibodies commercially available for SARS-CoV-1 proteins can also target SARS-CoV-2 proteins. IgM and IgG immunogenic epitopes of SARS-CoV-2 proteins were profiled in the serum of ten COVID-19 patients and can provide insight into the immune response to COVID-19 and are potential targets for COVID-19 diagnosis and vaccine development. Serological antibodies that may neutralize viral entry into host cells via the ACE2 receptor were identified.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Clinical characteristics of the recovered COVID-19 patients with re-detectable positive RNA test</div>
			<div class="collapsible-body"><strong>An</strong> et al. 3/30/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.26.20044222v1">link</a>]<br><strong>Summary — </strong>It has been reported that several cases recovered from COVID-19 tested positive for SARS-CoV-2 after discharge (re-detectable positive, RP). Our results showed that young and mild COVID-19 patients seem to be RP patients after discharge, who show no obviously clinical symptoms and disease progression upon re-admission.  RP and NRP patients displayed similar levels of IgG. The underlying mechanisms underlying RP occurrence remain unclear. 
and IgM in the plasma.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Neutralizing antibody responses to SARS-CoV-2 in a COVID-19 recovered patient cohort and their implications</div>
			<div class="collapsible-body"><strong>Wu</strong> et al. 4/20/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.30.20047365v2">link</a>]<br><strong>Summary — </strong>This study documents variations of SARS-CoV-2 specific neutralizing antibodies (NAbs) in recovered COVID-19 patients that may raise the concern about the role of NAbs on disease progression. The correlation of NAb titers with age, lymphocyte counts, and blood CRP levels suggested that the interplay between virus and host immune response in coronavirus infections should be further explored for the development of effective vaccine against SARS-CoV-2 virus.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Saliva is more sensitive for SARS-CoV-2 detection in COVID-19 patients than nasopharyngeal swabs</div>
			<div class="collapsible-body"><strong>Wyllie</strong> et al. 4/22/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.04.16.20067835v1">link</a>]<br><strong>Summary — </strong>The current gold standard for COVID-19 diagnosis is real-time                    
RT-PCR detection of SARS-CoV-2 from nasopharyngeal swabs. Our findings demonstrate that saliva is a viable and more sensitive alternative to nasopharyngeal swabs and could enable at-home                        
self-administered sample collection for accurate large-scale SARS-CoV-2 testing. </div></li></li>
	</ul>
<h4>Transmission of SARS-CoV-2</h4>
	<ul class="collapsible expandable">
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Molecular and serological investigation of 2019-nCoV infected patients: implication of multiple shedding routes</div>
			<div class="collapsible-body"><strong>Zhang</strong> et al. 2/17/2020 [<a href="https://www.tandfonline.com/doi/full/10.1080/22221751.2020.1729071">link</a>]<br><strong>Summary — </strong>Testing for 2019-nCoV infection is via oral swabs - assuming the virus was transmitted through the respiratory tract. We found presence of 2019-nCoV in anal swabs and blood as well, and more anal swab positives than oral swab positives in a later stage of infection, suggesting shedding and thereby transmitted through oral–fecal route. 

With respect to serology, we also showed serology test can improve detection positive rate thus should be used in future epidemiology. On day 5, an increase of viral antibodies can be seen in nearly all patients.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Virological assessment of hospitalized patients with COVID-2019</div>
			<div class="collapsible-body"><strong>Wolfel</strong> et al. 4/1/2020 [<a href="https://www.nature.com/articles/s41586-020-2196-x">link</a>]<br><strong>Summary — </strong>This paper provides proof of active virus replication in upper respiratory tract tissues. Infectious virus was readily isolated from throat- and lung-derived samples, but not from stool samples. Blood and urine never yielded virus. Shedding of viral RNA from sputum outlasted the end of symptoms.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Viral Kinetics and Antibody Responses in Patients with COVID-19</div>
			<div class="collapsible-body"><strong>Tan</strong> et al. 3/26/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.24.20042382v1">link</a>]<br><strong>Summary — </strong>Nasopharyngeal, sputum and stools rather than blood and urine, were the major shedding routes for SARS-CoV-2, and meanwhile sputum had a prolonged viral shedding. Symptom cough seems to be aligned with viral shedding in clinical respiratory and fecal specimens. Stronger antibody response was associated with delayed viral clearance and disease severity.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Presymptomatic SARS-CoV-2 Infections and Transmission in a Skilled Nursing Facility</div>
			<div class="collapsible-body"><strong>Arons</strong> et al. 4/24/2020 [<a href="https://www.nejm.org/doi/full/10.1056/NEJMoa2008457?query=recirc_top_ribbon_article_3">link</a>]<br><strong>Summary — </strong>Rapid and widespread transmission of SARS-CoV-2 was demonstrated in this skilled nursing facility. More than half of residents with positive test results were asymptomatic at the time of testing and most likely contributed to transmission. Infection-control strategies focused solely on symptomatic residents were not sufficient to prevent transmission after SARS-CoV-2 introduction into this facility.</div></li></li>
	</ul>
<h4>Serological test development & validation</h4>
	<ul class="collapsible expandable">
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Development and Clinical Application of A Rapid IgM‐IgG Combined Antibody Test for SARS‐CoV‐2 Infection Diagnosis</div>
			<div class="collapsible-body"><strong>Li</strong> et al. 2/27/2020 [<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/jmv.25727">link</a>]<br><strong>Summary — </strong>Diagnosis of SARS-CoV-2 infection with rtPCR is limited and produces high false-negative rates. We have developed a rapid and simple point‐of‐care lateral flow immunoassay that can detect immunoglobulin M (IgM) and IgG antibodies simultaneously against SARS‐CoV‐2 virus in human blood within 15 minutes which can detect patients at different infection stages.

Sensitivity: 88.66%
Specificity:  90.63%</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Serological immunochromatographic approach in diagnosis with SARS-CoV-2 infected COVID-19 patients</div>
			<div class="collapsible-body"><strong>Pan</strong> et al. 4/10/2020 [<a href="https://www.journalofinfection.com/article/S0163-4453(20)30175-4/fulltext">link</a>]<br><strong>Summary — </strong>We present the colloidal gold-based immunochromatographic (ICG) strip targeting viral IgM or IgG antibody. Sensitivity of ICG assay was 11.1%, 92.9% and 96.8% at the early stage (1–7 days after onset), intermediate stage (8–14 days after onset), and late stage (more than 15 days), respectively.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Evaluation of Enzyme-Linked Immunoassay and Colloidal Gold- Immunochromatographic Assay Kit for Detection of Novel Coronavirus (SARS-Cov-2) Causing an Outbreak of Pneumonia (COVID-19)</div>
			<div class="collapsible-body"><strong>Xiang</strong> et al. 3/1/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.02.27.20028787v1">link</a>]<br><strong>Summary — </strong>ELISA and colloidal gold-immunochromatographic assay (GICA) for specific IgM and IgG antibodies had sensitivities of 87.3% and 82.4%, with no significant difference. Specificity for both was 100% for the negative controls.  </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Severe Acute Respiratory Syndrome Coronavirus 2−Specific Antibody Responses in Coronavirus Disease 2019 Patients</div>
			<div class="collapsible-body"><strong>Okba</strong> et al. 4/8/2020 [<a href="https://wwwnc.cdc.gov/eid/article/26/7/20-0841_article">link</a>]<br><strong>Summary — </strong>We developed serologic assays for detection of SARS-CoV-2 antibodies.  The IgA-based ELISA showed higher sensitivity than the IgG-based ELISA, but the IgG ELISA showed higher specificity. Although the IgA and IgG assays can be used for serologic diagnosis, IgG is longer lived and thus is preferred for serosurveillance studies.

Most PCR-confirmed SARS-CoV-2–infected persons seroconverted by 2 weeks after disease onset</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Performance of VivaDiagTM COVID‐19 IgM/IgG Rapid Test is inadequate for diagnosis of COVID‐19 in acute patients referring to emergency room department</div>
			<div class="collapsible-body"><strong>Cassaniti</strong> et al. 3/30/2020 [<a href="https://onlinelibrary.wiley.com/doi/full/10.1002/jmv.25800">link</a>]<br><strong>Summary — </strong>In contrast with a previous study, VivaDiag COVID‐19 IgM/IgG Rapid Test lateral flowimmunoassay (LFIA) had extremely poor levels of sensitivity in acute patients enrolled from the emergency room (18.4%). Specificity was 91.7%, NPV was 26.2% and PPV was 87.5%.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Evaluation of Nucleocapsid and Spike Protein-based ELISAs for detecting antibodies against SARS-CoV-2</div>
			<div class="collapsible-body"><strong>Liu</strong> et al. 3/30/2020 [<a href="https://jcm.asm.org/content/early/2020/03/27/JCM.00461-20.long">link</a>]<br><strong>Summary — </strong>Two ELISA kits based on recombinant SARS-CoV-2 nucleocapsid protein and spike protein were used for detecting IgM and IgG antibodies, and their diagnostic feasibility was evaluated. ELISA has a high sensitivity, especially for the detection of serum samples from patients after 10 days post-disease onset, it can be an important supplementary method for COVID-19 diagnosis.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Diagnostic value and dynamic variance of serum antibody in coronavirus disease 2019</div>
			<div class="collapsible-body"><strong>Jin</strong> et al. 4/3/2020 [<a href="https://pubmed.ncbi.nlm.nih.gov/32251798/">link</a>]<br><strong>Summary — </strong>Viral serological testing is an effective means of diagnosis for SARS-CoV-2 infection. The positive rate and titer variance of IgG are higher than those of IgM in COVID-19.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Developing antibody tests for SARS-CoV-2</div>
			<div class="collapsible-body"><strong>Petherick</strong> et al. 4/4/2020 [<a href="https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30788-1/fulltext">link</a>]<br><strong>Summary — </strong>General article describing the need for antibody tests and different considerations that need to be taken into account when assessing antibody tests. It also describes how reinfection of SARS-CoV-2 is unlikely because the spike protein is highly conserved. 
</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Evaluation of a COVID-19 IgM and IgG rapid test; an efficient tool for assessment of past exposure to SARS-CoV-2</div>
			<div class="collapsible-body"><strong>Hoffman</strong> et al. 4/14/2020 [<a href="https://www.tandfonline.com/doi/full/10.1080/20008686.2020.1754538">link</a>]<br><strong>Summary — </strong>We evaluated a commercially available test developed for rapid (within 15 minutes) detection of SARS-CoV-2-specific IgM and IgG by 29 PCR-confirmed COVID-19 cases and 124 negative controls. The results revealed a sensitivity of 69% and 93.1% and specificity of 100% and 99.2% for IgM and IgG, respectively.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Significance of Serology Testing to Assist Timely Diagnosis of SARS-CoV-2 infections: Implication from a Family Cluster</div>
			<div class="collapsible-body"><strong>Xu</strong> et al. 4/14/2020 [<a href="https://www.tandfonline.com/doi/full/10.1080/22221751.2020.1752610">link</a>]<br><strong>Summary — </strong>A 6 person family cluster of SARS-CoV-2 infections were tested for infection with both a serological test and molecular assay. 5/6 were positive according to the serological test while only 2/6 were positive according to the molecular assay. Although WHO only recommends serology testing as a “second-line” test where molecular assays is not available, the utility of serology should be more emphasized in the COVID-19 pandemic.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">A preliminary study on serological assay for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in 238 admitted hospital patients</div>
			<div class="collapsible-body"><strong>Liu</strong> et al. 3/8/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.06.20031856v1">link</a>]<br><strong>Summary — </strong>We aimed to evaluate the diagnostic value of serological assay for SARS-CoV-2. 10 days after initial onset of symptoms, the antibody positive rates jumped to above 80% from less than 50% and diagnosis for viral infection should be majorly dependent on serological assay. Before then, nucleic acid test should be used.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Heat inactivation of serum interferes with the immunoanalysis of antibodies to SARS-CoV-2</div>
			<div class="collapsible-body"><strong>Hu</strong> et al. 3/16/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.12.20034231v1">link</a>]<br><strong>Summary — </strong>Heat inactivation, done prior to testing, significantly interferes with the levels of antibodies to SARS-CoV-2. 44.12% of the detected IgM levels were dropped below the cut-off value after heating, suggesting heat inactivation can lead to false-negative results of these samples. Therefore, heat inactivation prior to immunoanalysis is not recommended. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">A serological assay to detect SARS-CoV-2 seroconversion in humans</div>
			<div class="collapsible-body"><strong>Amanat</strong> et al. 4/16/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.17.20037713v2">link</a>]<br><strong>Summary — </strong>Here we describe a serological ELISA to detect seroconversion upon SARS-CoV-2 infection. The method is based on reactivity to the immunogenic S protein of the virus. We did not formally assess specificity and sensitivity of our assay since it might be implemented with different readouts, but our data suggests high sensitivity and specificity. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Antibody responses to SARS-CoV-2 in COVID-19 patients: the perspective application of serological tests in clinical practice</div>
			<div class="collapsible-body"><strong>Long</strong> et al. 3/20/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.03.18.20038018v1">link</a>]<br><strong>Summary — </strong>The positive rate for IgG reached 100% around 20 days after symptoms onset. The median day of seroconversion for both lgG and IgM was 13 days after symptoms onset. Seroconversion of IgM occurred at the same time, or earlier, or later than that of IgG. The criteria of IgG seroconversion and ≥ 4-fold increase in the IgG titers in sequential samples together diagnosed 82.9% (34/41) of the patients.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Evaluation of nine commercial SARS-CoV-2 immunoassays</div>
			<div class="collapsible-body"><strong>Lassauniere</strong> et al. 4/10/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.04.09.20056325v1">link</a>]<br><strong>Summary — </strong> This study evaluated the sensitivity and specificity of nine commercially available serological tests. These included three  ELISAs and six point-of-care (POC) lateral flow tests.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Rapid diagnosis of SARS-CoV-2 infection by detecting IgG and IgM antibodies with an immunochromatographic device: a prospective single-center study</div>
			<div class="collapsible-body"><strong>Garcia</strong> et al. 4/24/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.04.11.20062158v2">link</a>]<br><strong>Summary — </strong>The aim of our study was to analyze the diagnostic performance of rapid serologic tests in COVID-19 pneumonia patients. Our study shows that serologic rapid tests can be used as a complement of PCR to diagnose SARS-CoV-2 infection after 14 days from the onset of symptoms. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Analysis of SARS-CoV-2 Antibodies in COVID-19 Convalescent Plasma using a Coronavirus Antigen Microarray</div>
			<div class="collapsible-body"><strong>Assis</strong> et al. 4/17/2020 [<a href="https://www.biorxiv.org/content/10.1101/2020.04.15.043364v1">link</a>]<br><strong>Summary — </strong>We describe the validation of a coronavirus antigen microarray containing immunologically significant antigens from SARS-CoV-2, in addition to SARS-CoV, MERS-CoV, common human coronavirus strains, and other common respiratory viruses. A comparison of antibody profiles detected on the array from control sera collected prior to the SARS-CoV-2 pandemic versus convalescent blood specimens from virologically confirmed COVID-19 cases demonstrates complete discrimination of these two groups</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Test performance evaluation of SARS-CoV-2 serological assays</div>
			<div class="collapsible-body"><strong>Whitman</strong> et al. 4/24/2020 [<a href="https://www.dropbox.com/s/cd1628cau09288a/SARS-CoV-2_Serology_Manuscript.pdf?dl=0">link</a>]<br><strong>Summary — </strong>We conducted an evaluation of 10 lateral flow assays (LFAs) and two ELISAs to detect anti-SARS-CoV-2 antibodies.  the percent seropositive increased with time interval, peaking at 81.8-100.0% in samples taken >20 days after symptom onset. Test specificity ranged from 84.3-100.0% in pre-COVID-19 specimens.  IgM detection was more variable than IgG, and detection was highest when IgM and IgG results were combined.</div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">Saliva is more sensitive for SARS-CoV-2 detection in COVID-19 patients than nasopharyngeal swabs</div>
			<div class="collapsible-body"><strong>Wyllie</strong> et al. 4/22/2020 [<a href="https://www.medrxiv.org/content/10.1101/2020.04.16.20067835v1">link</a>]<br><strong>Summary — </strong>The current gold standard for COVID-19 diagnosis is real-time                    
RT-PCR detection of SARS-CoV-2 from nasopharyngeal swabs. Our findings demonstrate that saliva is a viable and more sensitive alternative to nasopharyngeal swabs and could enable at-home                        
self-administered sample collection for accurate large-scale SARS-CoV-2 testing. </div></li></li>
	</ul>
<h4>Other</h4>
	<ul class="collapsible expandable">
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_peer.png" height="16" width="16" style="margin-right: 10px;">Characterization of spike glycoprotein of SARS-CoV-2 on virus entry and its immune cross-reactivity with SARS-CoV</div>
			<div class="collapsible-body"><strong>Ou</strong> et al. 3/27/2020 [<a href="https://www.nature.com/articles/s41467-020-15562-9">link</a>]<br><strong>Summary — </strong>hACE2 is confirmed as the receptor for SARS-CoV-2. SARS-CoV-2 enters hACE2 cells mainly through endocytosis. PIKfyve, TPC2 and cathepsin L are critical for entry. SARS-CoV-2 S protein is less stable than SARS-CoV S. Recovery from one infection (SARS-CoV or SARS-CoV-2) is unlikely to protect against infection of the other. </div></li></li>
		<li>
			<div class="collapsible-header"><img src="assets/images/covid/li_prep.png" height="16" width="16" style="margin-right: 10px;">SARS-CoV-2 neutralizing serum antibodies in cats: a serological investigation</div>
			<div class="collapsible-body"><strong>Zhang</strong> et al. 4/3/2020 [<a href="https://www.biorxiv.org/content/10.1101/2020.04.01.021196v1">link</a>]<br><strong>Summary — </strong>Previous studies suggested cat could be a potential susceptible animal of SARS-CoV-2. We investigated the infection of SARS-CoV-2 in cats by detecting specific serum antibodies. 15 of 102 (14.7%) cat sera collected after the outbreak were positive for the receptor binding domain of SARS-CoV-2 by ELISA, demonstrating that SARS-CoV-2 has infected the cat population in Wuhan during the outbreak</div></li></li>
	</ul> -->


        <footer class="page-footer">
          <div class="container">
            <div class="row">
              <div class="col l6 s12">
                <h5 class="white-text">About us</h5>
                <p class="grey-text text-lighten-4">This website is maintained by the <a href="https://larremorelab.github.io">Larremore Lab</a> at the University of Colorado Boulder with collaborations in the <a href="https://www.hsph.harvard.edu/yonatan-grad/">Grad Lab</a> and the <a href="https://ccdd.hsph.harvard.edu/people/michael-mina/">Mina Lab</a> at the Harvard T.H. Chan School of Public Health. Web design by <a href="https://sam.zhang.fyi">Sam Zhang</a> using <a href="https://materializecss.com/">materialize</a>.
</p>
              </div>
              <div class="col l4 offset-l2 s12">
                <h5 class="white-text">Links</h5>
                <ul>
                    <li>
                        <a href="https://twitter.com/covid19testgrp?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">Follow @covid19testgrp</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
                    </li>
                    <li><a href="https://groups.google.com/forum/#covid19testgrp/join">Join the mailing list.</a></li>
                </ul>
              </div>
            </div>
          </div>
          <div class="footer-copyright">
            <div class="container">
            © 2020 The COVID-19 Test Group
            <!--<a class="grey-text text-lighten-4 right" href="#!">More Links</a>-->
            </div>
          </div>
        </footer>



        </div>

        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
        <script type="text/javascript">
          document.addEventListener('DOMContentLoaded', function() {
            var elems = document.querySelectorAll('.collapsible');
            var instances = M.Collapsible.init(elems, {accordion: false});
          });
        </script>
        <!-- Compiled and minified JavaScript -->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
        <link rel="stylesheet" href="/assets/css/covid/covid19testgroup.css">
    </body>
</html>
